tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2829,6 +2849,70 @@ <h2>Create a distance feature</h2>
         this.selectedExampleAndInference =
           this.selected.length > 0
           ? this.examplesAndInferences[this.selected[0]] : null;
+        requestAnimationFrame(()=> this.setAttributions());
+      },
+
+      /**
+       * Finds attribution features and sets the attribution object for the
+       * selected example.
+       */
+      setAttributions: function() {
+        const attrib = {};
+        const selectedData = this.visdata[this.selected[0]];
```

Comment:
stupid question: couldn't read the entire file but it roughly looks like only the first element in `selected` is used. Why is it using array for its type?

Reply:
Can you please provide what kind of structure `visdata` has in the code? thanks!

Reply:
selected is bound to facets-dive selected attribute, which is an array of indices of selected elements in facets-dive. in the what-if tool we only make use of the first selected element.

added more comments to visdata and selected definitions to specify structure.